internal/plugins/info.csv

```diff
@@ -56,6 +56,7 @@ couchbase               ,cache     ,Couchbase               ,4.12.0  ,community
 couchbase               ,processor ,Couchbase               ,4.11.0  ,community  ,n          ,n     ,n
 csv                     ,input     ,csv                     ,0.0.0   ,certified  ,n          ,n     ,n
 csv                     ,scanner   ,csv                     ,0.0.0   ,certified  ,n          ,y     ,y
+cypher                  ,output    ,cypher                  ,0.0.0   ,community  ,n          ,n     ,n
```

Comment:
We can probably safely set the version to `4.37.0`. I'm not sure what our process is going to look like for determining when to enable for cloud, maybe worth @vladoschreiner chiming in during the PR phase.

Reply:
I pinged in slack, seems like the process is initially no until the security team reviews... 🤷

Reply:
Set the version, thanks!